portfolio/src/main/webapp/index.html

```diff
@@ -80,74 +80,73 @@ <h3>Welcome to my portfolio.</h3>
+            <a href="https://github.com/kyracq" target="_blank"><i class="fab fa-github fa-2x"></i></a>
+            <a href="https://www.linkedin.com/in/kyra-a" target="_blank"><i class="fab fa-linkedin-in fa-2x"></i></a>
           </div>
-          <button class="btn" onClick="jobsNext()">next</button>
         </div>
       </div>
-      <div class="section-wrapper">
-        <h1 id="comments-header">Comments</h1>
-        <form action="/comments" method="POST" name="comment-form">
-          <div class="add-comment">
-            <textarea placeholder="Leave a comment" name="comment-input" required></textarea>
-            <input type="text" placeholder="Name" name="name-input" required>
-            <input type="submit">
-                  </div>
-        </form>
-        <label for="max-comments">Number of comments to show:</label>
-        <input type="number" id="max-comments" name="max-comments" min="1" max="20" value="1">
-        <button onClick="getComments()">Show comments</button>
-        <div id="comments-container"></div>
+    </div>
+    <div class="section-wrapper" id="education">
+      <h1 class="right-line">Education</h1>
+      <p>
+        I am Class of 2022 and study computer science in the Engineering School at Princeton.
+        Some of my classes have included Algorithms and Data Structures, Intro to Programming Systems,
+        and Operating Systems.
+        At school I'm the mentorship chair for Princeton Women in CS (PWiCS) and am also involved in
+        Envision, which
+        aims to empower future leaders to pioneer a brighter future through the prudent advancement of
+        technology.
+      </p>
+    </div>
+    <div class="section-wrapper" id="experience">
+      <h1 class="left-line">Work experience</h1>
+      <div class="experience-content">
+        <button class="btn" onClick="jobsBack()">back</button>
+        <div id="carousel-content">
+          <h2 id="job"></h2>
+          <p id="date"></p>
+          <div id="summary"></div>
+        </div>
+        <button class="btn" onClick="jobsNext()">next</button>
       </div>
-      <footer>
-        This site was made by Kyra Acquah as part of the Google STEP program.
-      </footer>
+    </div>
+    <div id="comment-section" class="section-wrapper">
+      <h1 id="comments-header">Comments</h1>
+      <form action="/comments" method="POST" name="comment-form">
+        <div class="add-comment">
+          <textarea placeholder="Leave a comment" name="comment-input" required></textarea>
+          <input type="text" placeholder="Name" name="name-input" required>
+          <input type="submit">
+        </div>
+      </form>
+      <label for="max-comments">Number of comments to show:</label>
+      <input type="number" id="max-comments" name="max-comments" min="1" max="20" value="5" onChange="getComments()">
+      <div id="comments-container"></div>
+      <button onClick="deleteAllComments()">Delete all comments</button>
```

Comment:
I haven't done this part of the walkthrough yet, so it's not clear to me (like it wouldn't be to someone outside STEP) - does this delete all comments for just the user, or does it actually clear them off the site and datastore? If the latter, that seems like it might be important and worth clarifying.